### PR TITLE
update fortran-src

### PR DESCRIPTION
**Pull request by raehik**

### CODE REVIEW

File: src/Language/Fortran/Extras.hs [hunk 1]
```diff
@@ -93,8 +93,8 @@ programFile options = do
 
 incFile :: FortranSrcRunOptions -> IO [Block A0]
 incFile options = do
-  (pfPath, pfContents, pfIncludes, _fVersion) <- unwrapFortranSrcOptions options
-  f77lIncIncludes pfIncludes pfPath pfContents
+  (pfPath, pfContents, _pfIncludes, _fVersion) <- unwrapFortranSrcOptions options
```
Comment (raehik) [inline src/Language/Fortran/Extras.hs:96]: @RaoulHC to confirm, was this the change you were expecting to make?

Comment (RaoulHC) [reply src/Language/Fortran/Extras.hs:96]: Indeed

**Comment by RaoulHC** [comment]
LGTM.

Seems like the CI is failing because there are some cabal build steps that aren't using the version specified in the stack file. Is there a way of getting cabal to use the right version?

**Comment by raehik** [comment]
Cabal is using the current Hackage version while Stack I pinned a commit so expected. I'll throw a quick fortran-src release together.

**Pull request merged**